public_html/src/userObject.php

```diff
@@ -56,13 +56,20 @@ public function __construct(array $vars, $fromDB){
 			$this->toolAdmin = 0;
 			$this->checkuser = 0;
 			$this->developer = 0;
-			$this->useSecure = isset($vars['useSecure']);
+			$this->useSecure = !isset($vars['useSecure']);

```

Comment:
I'm not sure what consequences come from this change for users to check whether or not to use secure. Either way it needs to be backwards compatible.